src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -30,7 +24,7 @@
     float: left;
     background-image: file-url("separator.png");
 
-    @include device-pixel-ratio() {
+    @include govuk-h-device-pixel-ratio() {
```

Comment:
Linter is complaining about the empty-args. We can update this line (and the other one is `_buttons.scss`) or we can adjust the linter if we're not happy with him picking that up.

Reply:
I think we should remove the empty parentheses, but I think it makes more sense to fix existing linting issues in another PR.